src/BuildingBlocks/EventBus/EventBusServiceBus/EventBusServiceBus.cs

```diff
@@ -125,25 +130,28 @@ public void Dispose()
 
         private void RegisterSubscriptionClientMessageHandler()
         {
-            _serviceBusPersisterConnection.SubscriptionClient.RegisterMessageHandler(
-                async (message, token) =>
+            ServiceBusProcessorOptions options = new ServiceBusProcessorOptions { MaxConcurrentCalls = 10, AutoCompleteMessages = false };
+            ServiceBusProcessor processor = _serviceBusPersisterConnection.TopicClient.CreateProcessor(_topicName, options);
```

Comment:
I don't see where we are actually starting the processor. We would also need to cache it in a class member so that it can be closed (I think probably in the Dispose method).